### DISCUSSION TITLE
Disable visual bell through wezterm configuration

### DISCUSSION THREAD

**Discussion by lminer** [wezterm/wezterm#5960] [Q&A]
### What Operating System(s) are you running on?

macOS

### WezTerm version

wezterm 20240203-110809-5046fc22

### Ask your question!

Tab completion is causing an awful visual bell on wezterm. I managed to suppress it by adding `unsetopt BEEP` in my .zshrc. Unfortunately, I have to do the same on every remote machine that I ssh into, which isn't really feasible. Is there any way to tell wezterm just to ignore the visual bell no matter what?

**Comment by wez** [comment]
https://wezfurlong.org/wezterm/config/lua/config/visual_bell.html

**Comment by coryfklein** [comment]
For other Googlers like me, the relevant section from the above linked docs says:

> If the total fade in and out durations are 0, then there will be no visual bell indication.

So this config will disable the visual bell:

```
config.visual_bell = {
  fade_in_duration_ms = 0,
  fade_out_duration_ms = 0,
}
```
